pkg/operator/connectivitycheckcontroller/manifests/controlplane.operator.openshift.io_podnetworkconnectivitychecks.yaml

```diff
@@ -41,10 +41,10 @@ spec:
           metadata:
             type: object
           spec:
-            description: Spec defines the source and target of the connectivity check
+            description: spec defines the source and target of the connectivity check
```

Comment:
why has this file been changed ? 

if this file was changed because of `make update` command then please add this change to a separate commit.

Reply:
other than that it LGTM

Reply:
Ok, will do. thanks!